src/main/app/forms/models/otherDamages.ts

```diff
@@ -1,14 +1,16 @@
 export class OtherDamages {
-  static readonly LESS = new OtherDamages('LESS', 'less')
-  static readonly MORE = new OtherDamages('MORE', 'more')
-  static readonly NONE = new OtherDamages('NONE', 'none')
+  static readonly LESS = new OtherDamages('LESS', 'less', 'thousandPoundsOrLess')
```

Comment:
why do you need an extra variable here? I have a PR pending that is turning this back into just a string rather than an object. is the object needed for the claim store?